src/sourcetransform.rs

```diff
@@ -1,18 +1,23 @@
-use crate::error::Error::SourceTransformerError;
-use crate::error::ErrorKind::UserDefinedError;
-use crate::shared::{self, prost_timestamp_from_utc};
+use crate::error::Error::{self, SourceTransformerError};
+use crate::error::ErrorKind;
+use crate::shared::{self, prost_timestamp_from_utc, utc_from_timestamp};
 use chrono::{DateTime, Utc};
+use proto::SourceTransformResponse;
 use std::collections::HashMap;
 use std::fs;
 use std::path::PathBuf;
 use std::sync::Arc;
 use tokio::sync::{mpsc, oneshot};
+use tokio::task::JoinHandle;
+use tokio_stream::wrappers::ReceiverStream;
 use tokio_util::sync::CancellationToken;
-use tonic::{async_trait, Request, Response, Status};
+use tonic::{async_trait, Request, Response, Status, Streaming};
+use tracing::{error, info};
 
 const DEFAULT_MAX_MESSAGE_SIZE: usize = 64 * 1024 * 1024;
 const DEFAULT_SOCK_ADDR: &str = "/var/run/numaflow/sourcetransform.sock";
 const DEFAULT_SERVER_INFO_FILE: &str = "/var/run/numaflow/sourcetransformer-server-info";
+const DEFAULT_CHANNEL_SIZE: usize = 1000;
 
 const DROP: &str = "U+005C__DROP__";
 
```

Comment:
`pub mod proto` rename `proto` to `sourcetransformer_pb` ?

Reply:
I don't think that is needed since the it's absolute name (`crate::sourcetransform::proto`) already makes the namespacing clear.

Reply:
@yhl25 had some opinions on this.